pkg/networks/sudoers.go

```diff
@@ -0,0 +1,93 @@
+package networks
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func Sudoers() (string, error) {
+	config, err := Config()
+	if err != nil {
+		return "", err
+	}
+
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("%%%s ALL=(root:wheel) NOPASSWD:NOSETENV: %s\n", config.Group, config.MkdirCmd()))
+
+	// names must be in stable order to be able to check if sudoers file needs updating
+	names := make([]string, 0, len(config.Networks))
+	for name := range config.Networks {
+		names = append(names, name)
+	}
+	sort.Strings(names)
+
+	for _, name := range names {
+		sb.WriteRune('\n')
+		sb.WriteString(fmt.Sprintf("# Manage %q network daemons\n", name))
+		for _, daemon := range []string{Switch, VMNet} {
+			sb.WriteRune('\n')
+			sb.WriteString(fmt.Sprintf("%%%s ALL=(%s:%s) NOPASSWD:NOSETENV: \\\n",
+				config.Group, config.DaemonUser(daemon), config.DaemonGroup(daemon)))
+			sb.WriteString(fmt.Sprintf("    %s, \\\n", config.StartCmd(name, daemon)))
+			sb.WriteString(fmt.Sprintf("    %s\n", config.StopCmd(name, daemon)))
+		}
+	}
+	return sb.String(), nil
```

Comment:
We should have unit tests to validate the generated strings.
Can be another PR, though.

Reply:
> We should have unit tests to validate the generated strings.

Added in https://github.com/lima-vm/lima/pull/239/commits/42fcd35d4ad0bee35253f089d728d20f7a9f42d4